index.html

```diff
@@ -15,5 +15,89 @@
 
 <body>
 <!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+<div class='container'>
+
+  
+  
+  <nav class="navigation">
+    <div class='aNav'>
+      <a class='a1' href='services.html'>Service</a>
+      <a href=''>Product</a>
+      <a href=''>Vision</a>
+      <a href=''>Features</a>
+      <a href=''>About</a>
+      <a href=''>Contact</a>
+    </div>
+    <div class='logoDiv'>
```

Comment:
This div is unnecessary as you only have one element inside of it. You can just target the image tag specifically in your CSS if you want to adjust styles for it :)